e2e/testgames/game/Dockerfile

```diff
@@ -17,12 +17,36 @@ RUN cd e2e/testgames/game && go build -v -o /go/bin/app
 
 
 ################################
-# Runtime Image
+# Runtime Image - Normal
 ################################
 FROM gcr.io/distroless/base-debian12 AS runtime
 
 # Copy the binary from the build image
 COPY --from=build /go/bin/app /usr/bin
 
 # Run the binary
-CMD ["app"]
\ No newline at end of file
+CMD ["app"]
+
+################################
+# Runtime Image - Debug
+################################
+FROM golang:1.22-bookworm AS runtime-debug
+
+WORKDIR /go/src/app
+
+COPY e2e/testgames e2e/testgames
+COPY cardinal cardinal
+COPY assert assert
+
+# Install delve
+RUN go install github.com/go-delve/delve/cmd/dlv@latest
+
+RUN cd e2e/testgames && go mod download
```

Comment:
This should be moved to right after `COPY e2e/testgames e2e/testgames` so that we dont have to redo the go mod download as long as `e2e/testgames` is unchanged; especially useful since go mod download can be a bit slow...